CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+Version 1.14.2
+==============
+
+This release adds support for Bokeh 2.3, introduces a number of minor
+enhancements, miscellaneous documentation improvements and a good number
+of bug fixes.
+
+Many thanks to the many contributors to this release, whether directly
+by submitting PRs or by reporting issues and making
+suggestions. Specifically, we would like to thank @philippjfr for the
+Bokeh 2.3 compatibility updates, @kcpevey, @timgates42, and @scottstanie
+for documentation improvements as well as @Hoxbro and @LunarLanding for
+various bug fixes. In addition, thanks to the maintainers @jbednar,
+@jlstevens and @philippjfr for contributing to this release.
+
+Enhancements:
+
+- Bokeh 2.3 compatibility
+  ([#4805](https://github.com/holoviz/holoviews/pull/4805),
+  [#4809](https://github.com/holoviz/holoviews/pull/4809))
+- Supporting dictionary streams parameter in DynamicMaps and operations
+  ([#4787](https://github.com/holoviz/holoviews/pull/4787),
+   [#4818](https://github.com/holoviz/holoviews/pull/4818),
+   [#4822](https://github.com/holoviz/holoviews/pull/4822))
+- Added inspect datashader operation supporting points and polygons
+  ([#4794](https://github.com/holoviz/holoviews/pull/4794))
+- Support spatialpandas DaskGeoDataFrame
+  ([#4792](https://github.com/holoviz/holoviews/pull/4792))
+- Disable zoom on axis for geographic plots
+  ([#4812](https://github.com/holoviz/holoviews/pull/4812)
+- Fix Area stack classmethod for non-aligned data
+  ([#4836](https://github.com/holoviz/holoviews/pull/4836))
```

Comment:
That sounds like a bugfix, not an enhancement. Would it be accurate to say it like an enhancement:

```suggestion
- Add support for non-aligned data in Area stack classmethod
  ([#4836](https://github.com/holoviz/holoviews/pull/4836))
```

Otherwise this should presumably move to the bugfix section.

Reply:
Good idea.